### PR TITLE
Add Date Formatting to guidelines

### LINKED ISSUE #50: Standardize preferred date/time format in docs guidelines
We've used non-technical date and time formats (used to explain e.g. important dates admins need to be aware of) somewhat inconsistently in the docs (e.g., "30 April" vs. "April 30"). Let's standardize, and update the guidelines. [AP style](https://coschedule.com/blog/ap-style-cheat-sheet/#days-dates-years) seems like a good fit for the English language docs as it's unambiguous and more user-friendly than ISO dates (`2019-02-26`) for this purpose.

*(Edit 2020-10-20)* Usage is now largely consistent, but we should still document it as part of the docs guidelines.

### PR DESCRIPTION
**Pull request by joaedwar**
## Status

Ready for review 

## Description of Changes

* Description: Add formatting guidelines for dates in docs

* Fixes #50 

## Testing

*

## Release 

* 

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

**Comment by joaedwar** [comment]
@eloquence

File: LICENSE [hunk 1]
```diff
@@ -2,7 +2,7 @@ Copyright (C) 2013- Aaron Swartz, James Dolan, Freedom of the Press Foundation,
 and SecureDrop contributors
 
                     GNU AFFERO GENERAL PUBLIC LICENSE
-                       Version 3, 19 November 2007
```
Comment (eloquence) [inline LICENSE:5]: The license is a third party asset and should not be modified.

**Review by eloquence — CHANGES_REQUESTED** [review]
One requested change, otherwise LGTM :). Thanks @joaedwar

**Comment by joaedwar** [comment]
@eloquence was this changed back?

**Comment by eloquence** [comment]
@joaedwar What do you mean? I've not pushed to your branch, if that's what you're asking. The pending change (to restore the license to its original state) is still pending.

**Review by eloquence — APPROVED** [review]
I've reverted the LICENSE file change and updated your branch accordingly; this is now good to go, so merging.

**Pull request merged**